src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -591,6 +591,27 @@ private void ImportModule_ViaAssembly(ImportModuleOptions importModuleOptions, A
             }
         }
 
+        private PSModuleInfo ImportModule_LocallyViaName_Wrapper(ImportModuleOptions importModuleOptions, string name)
```

Comment:
The name of this method doesn't really describe what the added functionality is. Perhaps something like `ImportModule_LocallyViaName_WithTelemetry` or `ImportModule_LocallyViaName_ForWinCompat`?

Reply:
It's hard to tell from the diff, but this is actually a cut-and-paste from existing code. I had to resuse it, so moved it out to this separate function. Didn't come up with anything better than `_Wrapper`.  Though `_WithTelemetry` is probably better; updated.